user_delete.php

```diff
@@ -0,0 +1,11 @@
+<?php
+require "db_connection.php";
+
+$user_id = $_POST['user_id'];
+
+$sql = 'DELETE FROM users WHERE id = :id';
```

Comment:
```suggestion
$sql = 'update users set delete_flag = 1, deleted_at = current_timestamp() WHERE id = :id';
```
usersテーブルは物理削除でなく論理削除になります。